client/blocks/login/continue-as-user.scss

```diff
@@ -2,7 +2,8 @@
 @import "@wordpress/base-styles/breakpoints";
 
 .continue-as-user {
-	display: block;
+	display: flex;
+	flex-direction: column;
```

Comment:
Does flex make a difference here vs block? Visually these sections seem identical in all the variants I checked. I suppose we could use it to leverage gap instead of margin for spacing.

Reply:
This might be only on the screens you couldn't reach; it makes the inner divs not fill 100% of the horizontal space, so the layout is closer to the white login version.

With flex:

![Screenshot 2025-05-19 at 2 23 02 PM](https://github.com/user-attachments/assets/d221b285-4756-4ceb-bf71-f97237533f2f)

Without flex:

![Screenshot 2025-05-19 at 2 19 43 PM](https://github.com/user-attachments/assets/8b39faa3-9e27-4d98-a30c-77a78e95116b)

Reply:
Ok, I can see it now!